service/web/tabs/app/src/containers/CloneFormContainer.tsx

```diff
@@ -0,0 +1,365 @@
+import * as React from "react"
+import { connect } from "react-redux"
+import Axios from "axios"
+import {
+  IDispatchProps,
+  IState,
+  mapDispatchToProps,
+  mapStateToProps
+} from "../ducks"
+import {
+  H1,
+  FormGroup,
+  NumericInput,
+  InputGroup,
+  Intent,
+  Checkbox,
+  Button,
+  H2,
+  H5,
+  Tooltip,
+  Classes,
+  Spinner,
+  Radio,
+  RadioGroup
+} from "@blueprintjs/core"
+import { FlexContainer } from "@misk/core"
+import { Link } from "react-router-dom"
+import { FormEvent } from "react"
+import { IBackfill } from "../components"
+import { LayoutContainer } from "../containers"
+
+interface CloneFormState {
+  loading: boolean
+  errorText?: string
+
+  statusResponse: any
+
+  backfills: IBackfill[]
+  backfill?: IBackfill
+  dry_run: boolean
+
+  scan_size: number
+  batch_size: number
+  num_threads: number
+  range_clone_type: string
+  pkey_range_start?: string
+  pkey_range_end?: string
+  extra_sleep_ms: number
+  backoff_schedule?: string
+  parameters: any
+}
+
+class CloneFormContainer extends React.Component<
+  IState & IDispatchProps,
+  IState & CloneFormState
+> {
+  private id: string = (this.props as any).match.params.id
+
+  componentDidMount() {
+    this.setState({
+      loading: false,
+      errorText: null,
+
+      statusResponse: null,
+
+      backfill: null,
+
+      dry_run: false,
+      scan_size: 10000,
+      batch_size: 100,
+      num_threads: 1,
+      range_clone_type: "RESTART",
+      pkey_range_start: null,
+      pkey_range_end: null,
+      backoff_schedule: null,
+      extra_sleep_ms: 0,
+      parameters: {}
+    })
+    Axios.get(`/backfills/${this.id}/status`)
```

Comment:
axios is easy, I'll probably replace all the http with this

Reply:
I'll have to look this up.